paddle/operators/scatter.h

```diff
@@ -0,0 +1,92 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
It looks like most content of this PR duplicates with https://github.com/PaddlePaddle/Paddle/pull/3371